test/REGRESSIONS

```diff
@@ -426,6 +410,10 @@ Reviewed 2015-01-23
 
 === sporadic failures below ===
 
+PGC-W-0205-Argument mismatch for assert - all but a few failed
+--------------------------------------------------------------
+(xe-wb.prgenv-pgi 2015-01-29)
```

Comment:
If this is xe-wb.prgenv-pgi-specific, shouldn't/couldn't it be further down in the file (like in the pgi section)?

Also, why are the configuration/dates listed before the line (where I'd expect only tests to be listed) rather than above it?